check_docker

```diff
@@ -396,7 +404,9 @@ def check_cpu(container, warn, crit, _=''):
         max = 100
         evaluate_numeric_thresholds(container=container, value=usage, warn=warn, crit=crit, units='%', name='cpu',
                                     short_name='cpu', min=0, max=max)
-
+    else:
```

Comment:
humm... tried to fix that, but I'm running into issues: as soon as I add `get_container_info(container)['State']` in the `perform_checks` function, I get errors on all tests:

```
FAIL: test_check_cpu (__main__.TestPerform)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "test_check_docker.py", line 963, in test_check_cpu
    self.assertEqual(patched.call_count, 1)
AssertionError: 0 != 1

Stdout:
FOO; BAR; UNKNOWN: Exception raised during check: TypeError('list indices must be integers or slices, not str',)|1;2;3;4;
```

I'm not that familiar with unittest,and I can't get the full stacktrace to pinpoint the issue.
Also, it's working fine with my real docker instance!

I'll push a WIP commit so you can take a look.